images/local-storage-mustgather.yml

```diff
@@ -23,6 +23,7 @@ enabled_repos:
 for_payload: false
 from:
   member: openshift-enterprise-cli
-name: openshift/ose-local-storage-mustgather
```

Comment:
prior to 4.14 this was not an operand so it didn't matter that it wasn't published to the matching CDN. now it is so we need to match where it's actually published.